sci-libs/hdf5/hdf5-1.10.1.recipe

```diff
@@ -72,15 +75,15 @@ PROVIDES_devel="
 	devel:libdynlibdump$secondaryArchSuffix
 	devel:libdynlibls$secondaryArchSuffix
 	devel:libdynlibvers$secondaryArchSuffix
-	devel:libhdf5$secondaryArchSuffix = 10.2.1 compat >= 10
-	devel:libhdf5_cpp$secondaryArchSuffix = 13.0.0 compat >= 13
-	devel:libhdf5_fortran$secondaryArchSuffix = 10.0.4 compat >= 0
-	devel:libhdf5_hl$secondaryArchSuffix = 10.1.1 compat >= 10
-	devel:libhdf5_hl_cpp$secondaryArchSuffix = 11.1.0 compat >= 11
-	devel:libhdf5hl_fortran$secondaryArchSuffix = 10.0.3 compat >= 10
+	devel:libhdf5$secondaryArchSuffix = 101.0.0 compat >= 101
+	devel:libhdf5_cpp$secondaryArchSuffix = 101.0.0 compat >= 13
+	devel:libhdf5_fortran$secondaryArchSuffix = 100.1.0 compat >= 0
+	devel:libhdf5_hl$secondaryArchSuffix = 100.0.1 compat >= 10
+	devel:libhdf5_hl_cpp$secondaryArchSuffix = 100.1.0 compat >= 11
+	devel:libhdf5hl_fortran$secondaryArchSuffix = 100.0.1 compat >= 10
```

Comment:
I think you forgot to adjust these `compat >=`

Reply:
I wanted to ask, if somebody knows where those numbers coming from. I don't want to cascade-rev-bump everything if not needed. Soname is still black magic to me.

Reply:
You can get the compat number via `readelf -d <path to lib>`, there will be something like this appears:

     0x000000000000000e (SONAME)             Library soname: [libncursesw.so.6]

Break down:
- library name: libncursesw
- compat version: 6